### ISSUE TITLE
Remove puli.phar

### ISSUE THREAD

**Issue by sagikazarmark** [php-http/discovery#48]
Once puli/issues#165 is done

**Comment by sagikazarmark** [comment]
Now that Puli is optional, should we remove the binary from the package? Would it be a BC break if we did it later?

**Comment by Nyholm** [comment]
I think we should remove it. 
It would never be a BC break beacuse the discovery strategies are not part of the BC promise. IE we could remove the PuliStrategy without breaking BC....

This is a bit weird when I think about it... I'll document it. But yes, Puli.phar should be removed before 1.0.

**Comment by dbu** [comment]
fixed in #76
